src/main/java/org/hydev/mcpm/client/arguments/CommandsFactory.java

```diff
@@ -31,14 +26,15 @@ private CommandsFactory() { }
      */
     public static List<CommandParser> baseParsers() {
         var echoController = new EchoController();
+        var exportPluginsController = new ExportPluginsController(null); // do I have to instantiate the entire thing ??
 
         /*
          * Add general parsers to this list!
          * All versions of MCPM will have access to these parsers.
          * If you're not sure if your command is server-only, add it to this list!
          */
         return List.of(
-            new EchoParser(echoController)
+            new EchoParser(echoController), new ExportPluginsParser(exportPluginsController)
```

Comment:
I mean... if I'm being pedantic...

```suggestion
            new EchoParser(echoController),
            new ExportPluginsParser(exportPluginsController)
```